docs/man/git-lfs-ext.1.ronn

```diff
@@ -0,0 +1,25 @@
+git-lfs-ext(1) - View extension details
+==============================================================
+
+## SYNOPSIS
+
+`git lfs ext list` [<name>...]
+
+## DESCRIPTION
+
+Git LFS extensions enable the manipulation of files streams
+during smudge and clean.
+
+## EXAMPLES
+
+* List details for all extensions
+
+    `git lfs ext list`
+
+* List details for the specified extensions
+
+    `git lfs ext list 'foo' 'bar'`
+
```

Comment:
I believe the current implementation looks like this now:

```
# list all
git lfs ext

# list all
git lfs ext list

# list some
git lfs ext list "foo"
```

Totally cool :+1: Part of my question was haggling on the difference between `list` and `show`. But since there are no extension details, I think `list` is fine. This is how `git remote` works too:

```
$ git remote
origin

$ git remote show
origin

$ git remote show origin
* remote origin
  Fetch URL: https://github.com/github/git-lfs
  Push  URL: https://github.com/github/git-lfs
  HEAD branch: master
  Remote branches:
...
```